components/Profile/TeamCard.js

```diff
@@ -11,6 +11,7 @@ export default function TeamCard({ team }) {
                         src={team.hackathon.image}
                         style={{ height: 200 }}
                         rounded={{ t: "xl" }}
+                        alt="Hackathon Logo"
```

Comment:
We can use the hackathon title here with this text like `team.hackathon.title + " Logo"`

Reply:
The required changes are applied